unabridged-works-volume-1/original/07-axe-to-the-root.adoc

```diff
@@ -536,15 +548,17 @@ and a new one formed,
 which alone is able to receive and retain the new wine of the kingdom; whereas the other,
 Pharisee-like, can only receive a relation of the letter concerning the kingdom.
 
-=== SOME ASSERTIONS
+=== Some Assertions
 
+[.section-summary-preface]
 Concerning FAITH, its Nature, Rise, etc., with its Receiving of CHRIST,
```

Comment:
Another perfect usage of [.section-summary-preface]. 👍

But, can you switch FAITH and CHRIST here to italics (with first-letter capitalized, as with the above comment on 299)?  Thanks!